models.py

```diff
@@ -1,6 +1,35 @@
 from room import db
+from flask_login import UserMixin
+from werkzeug.security import generate_password_hash, check_password_hash
+from room import login
+@login.user_loader
+def load_user(id):
+    return User.query.get(int(id))
 
+class ConciergeMii(db.Model):
+     mii_id = db.Column(db.Integer, primary_key=True, unique=True)
+     title = db.Column(db.String(6))
+     color1 = db.Column(db.String(6))
+     color2 = db.Column(db.String(6))
+     message1 = db.Column(db.String(100))
+     message2 = db.Column(db.String(100))
+     message3 = db.Column(db.String(100))
+     message4 = db.Column(db.String(100))
+     message5 = db.Column(db.String(100))
+     message6 = db.Column(db.String(100))
+     message7 = db.Column(db.String(100))
+     updated = db.Column(db.String(17))
+     movieid = db.Column(db.String(1))
+class User(db.Model, UserMixin):
+    # Used to login to the Admin Panel
+    id = db.Column(db.Integer, primary_key=True,default=1)
+    username = db.Column(db.String(100))
+    password_hash = db.Column(db.String(9999999))
+    def set_password(self, password):
+        self.password_hash = generate_password_hash(password)
```

Comment:
It seems like Flask uses PBKDF2 here which is long but not 9999999 long - perhaps it'd be better to just use `db.String` itself so that databases can determine how to optimise storage?